server/controllers/finance/reports/financial.employee.handlebars

```diff
@@ -0,0 +1,64 @@
+{{> head title="REPORT.EMPLOYEE_STANDING.TITLE" }}
+
+<body>
+
+  {{> header }}
+
+  <h3 class="text-center text-uppercase"><strong>{{translate "REPORT.EMPLOYEE_STANDING.TITLE"}}</strong></h3>
+  <h3 class="text-center"><strong>{{ employee.code }} | {{ employee.display_name }}</strong></h3>
+
+  <section>
+    <table class="table table-condensed table-bordered table-report">
+      <thead>
+        <tr class="text-capitalize text-center" style="background-color: #ddd;" >
+          <th>{{translate "TABLE.COLUMNS.DATE" }}</th>
+          <th style="width: 10%;">{{translate "TABLE.COLUMNS.DOCUMENT" }}</th>
+          <th style="width: 5%;">{{translate "TABLE.COLUMNS.TRANSACTION" }}</th>
+          <th>{{translate "TABLE.COLUMNS.DESCRIPTION" }}</th>
+          <th style="width: 15%;" class="text-center">{{translate "TABLE.COLUMNS.AMOUNT" }}</th>
+          <th style="width: 15%;" class="text-center">{{translate "TABLE.COLUMNS.BALANCE" }}</th>
+        </tr>
+      </thead>
+      <tbody>
+        {{#each transactions}}
+          <tr>
+            <td>{{date this.trans_date}}</td>
+            <td class="text-right" title="{{this.document}}">{{this.document}}</td>
+            <td class="text-right" title="{{this.trans_id}}">{{this.trans_id}}</td>
+            <td class="text-uppercase" style="max-width : 200px; white-space : nowrap; overflow : hidden; text-overflow : ellipsis;" title="{{this.description}}">{{this.description}}</td>
+            <td class="text-right">
+              {{debcred this.balance ../metadata.enterprise.currency_id}}
+            </td>
+            <td class="text-right">
+              {{debcred this.cumsum ../metadata.enterprise.currency_id}}
+            </td>
+          </tr>
+        {{else}}
+          {{>emptyTable columns=5}}
```

Comment:
The `columns` attribute should match the number of `<td>`s in the `{{#each}}` section.  It looks like this should be 6.